passenger-service/src/main/java/by/shestakov/passengerservice/service/impl/PassengerServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package by.shestakov.passengerservice.service.impl;
+
+import by.shestakov.passengerservice.dto.request.PassengerRequest;
+import by.shestakov.passengerservice.dto.response.PassengerResponse;
+import by.shestakov.passengerservice.entity.Passenger;
+import by.shestakov.passengerservice.exception.PassengerAlreadyExistsException;
+import by.shestakov.passengerservice.exception.PassengerWasDeletedException;
+import by.shestakov.passengerservice.exception.PassengerNotFoundException;
+import by.shestakov.passengerservice.mapper.PassengerMapper;
+import by.shestakov.passengerservice.repository.PassengerRepository;
+import by.shestakov.passengerservice.service.PassengerService;
+import by.shestakov.passengerservice.util.ExceptionConstants;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+
+@RequiredArgsConstructor
+@Service
+public class PassengerServiceImpl implements PassengerService {
+
+    private final PassengerRepository passengerRepository;
+    private final PassengerMapper passengerMapper;
+
+    @Override
+    public List<PassengerResponse> getAllPassengers() {
+        return passengerRepository.findAll().stream().map(passengerMapper::toDto).toList();
```

Comment:
Стримы форматируются так:
```suggestion
        return passengerRepository.findAll().stream()
        .map(passengerMapper::toDto)
        .toList();
```